docs/keyd.scdoc

```diff
@@ -744,8 +744,8 @@ A key may optionally be bound to an _action_ which accepts zero or more argument
 		*overloadi(<key>, overloadt2(<layer>, <key>, <hold timeout>), <idle timeout>)*
 
 *timeout(<action 1>, <timeout>, <action 2>)*
-	If no key events occur within _<timeout> ms_, execute <action1>, otherwise
-	execute <action1>.
+	If no key events occur within _<timeout> ms_, execute <action 1>, otherwise
+	execute <action 2>.
```

Comment:
`action1` was present twice. I replaced the second one with `action2` but I actually never tested this feature, so I'm not sure it's the right way around. I'll test it as soon as I can.